test/utils/runners.go

```diff
@@ -218,6 +223,26 @@ func (config *DeploymentConfig) GetKind() schema.GroupKind {
 	return extensionsinternal.Kind("Deployment")
 }
 
+func (config *DeploymentConfig) GetClient() clientset.Interface {
```

Comment:
Again - if you are using method from underlying object, I think you don't need to redefine it.